docs/fuchsia.md

```diff
@@ -15,12 +15,28 @@ make extract TARGETOS=fuchsia SOURCEDIR=/path/to/fuchsia/checkout
 make generate
 ```
 
-To build binaries:
+Fuchsia generally does not support out-of-tree Go binary builds.
+
+However, syzkaller is mirrored in third_party/syzkaller.
+
+So, to build target binaries, run `fx full-build` in your SOURCEDIR.
```

Comment:
Is it this build file?
https://fuchsia.googlesource.com/garnet/+/master/testing/syzkaller/BUILD.gn
If yes, I don't think it will work because it does not define `github.com/google/syzkaller/sys.GitRevision` symbol. But it's only tip of the iceberg, we are basically tearing source project apart into pieces and commit these pieces into different repositories. For continuous testing we will need to update these parts in sync. I don't see how that will be possible.